manifest.yaml

```diff
@@ -32,17 +30,10 @@ main:
     db: /var/lib/postgresql/13
     dbconfig: /etc/postgresql/13
 health-checks:
-  main:
+  web-ui:
```

Comment:
this needs to match the name defined in the JS health check, which looks like "alive"

Reply:
changed to `alive`